metering/reports/metering-storage-locations.adoc

```diff
@@ -0,0 +1,76 @@
+[id="metering-storage-locations"]
+= Storage Locations
+include::modules/common-attributes.adoc[]
+:context: metering
+
+toc::[]
+
+A StorageLocation is a custom resource that configures where data will be stored by the reporting-operator. This includes the data collected from Prometheus, and the results produced by generating a Report custom resource.
```

Comment:
When introducing a new term, it needs to be italicized: `A _StorageLocation_ is a custom resource that...`

https://github.com/openshift/openshift-docs/blob/master/contributing_to_docs/doc_guidelines.adoc#quick-markup-reference